{"name": "使用Wrong order inform 提交退货单成功", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nBacktrace:\n\tOrdinal0 [0x006D7413+2389011]\n\tOrdinal0 [0x00669F61+1941345]\n\tOrdinal0 [0x0055C658+837208]\n\tOrdinal0 [0x005891DD+1020381]\n\tOrdinal0 [0x0058949B+1021083]\n\tOrdinal0 [0x005B6032+1204274]\n\tOrdinal0 [0x005A4194+1130900]\n\tOrdinal0 [0x005B4302+1196802]\n\tOrdinal0 [0x005A3F66+1130342]\n\tOrdinal0 [0x0057E546+976198]\n\tOrdinal0 [0x0057F456+980054]\n\tGetHandleVerifier [0x00889632+1727522]\n\tGetHandleVerifier [0x0093BA4D+2457661]\n\tGetHandleVerifier [0x0076EB81+569713]\n\tGetHandleVerifier [0x0076DD76+566118]\n\tOrdinal0 [0x00670B2B+1968939]\n\tOrdinal0 [0x00675988+1989000]\n\tOrdinal0 [0x00675A75+1989237]\n\tOrdinal0 [0x0067ECB1+2026673]\n\tBaseThreadInitThunk [0x75906739+25]\n\tRtlGetFullPathName_UEx [0x77408E7F+1215]\n\tRtlGetFullPathName_UEx [0x77408E4D+1165]", "trace": "self = <test_main.Test_Main object at 0x000001C4E1C918B0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"0acb60f99b608d3f06496c14e23dc941\")>\nParameter = {'assert_way': 'xpath=/html/body/div[2]/span/div/div/div/span', 'case_no': '', 'casename': '使用Wrong order inform 提交退货单...inform ation\\ntext=123412412\\nimg=C:\\\\Users\\\\admin\\\\PycharmProjects\\\\pythonProject\\\\okmarts_ui_test\\\\data\\\\1.jpg', ...}\n\n    @allure.feature('订单管理')\n    @pytest.mark.parametrize('Parameter', case_infos['my_order'], ids=casename_infos['my_order'])\n    def test_my_order(self, driver, Parameter):\n        allure.dynamic.title(Parameter['casename'])\n>       My_Order().my_order(driver,Parameter)\n\ntest_main.py:73: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\mode\\my_order.py:226: in my_order\n    WebDriverWait(driver, 20, 0.2).until(lambda x: x.find_element(by=f\"{assert_way.split('=', 1)[0]}\",\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"0acb60f99b608d3f06496c14e23dc941\")>\nmethod = <function My_Order.my_order.<locals>.<lambda> at 0x000001C4E1CBE430>\nmessage = ''\n\n    def until(self, method, message=''):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.time() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except InvalidSelectorException as e:\n                raise e\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, 'screen', None)\n                stacktrace = getattr(exc, 'stacktrace', None)\n            time.sleep(self._poll)\n            if time.time() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       Backtrace:\nE       \tOrdinal0 [0x006D7413+2389011]\nE       \tOrdinal0 [0x00669F61+1941345]\nE       \tOrdinal0 [0x0055C658+837208]\nE       \tOrdinal0 [0x005891DD+1020381]\nE       \tOrdinal0 [0x0058949B+1021083]\nE       \tOrdinal0 [0x005B6032+1204274]\nE       \tOrdinal0 [0x005A4194+1130900]\nE       \tOrdinal0 [0x005B4302+1196802]\nE       \tOrdinal0 [0x005A3F66+1130342]\nE       \tOrdinal0 [0x0057E546+976198]\nE       \tOrdinal0 [0x0057F456+980054]\nE       \tGetHandleVerifier [0x00889632+1727522]\nE       \tGetHandleVerifier [0x0093BA4D+2457661]\nE       \tGetHandleVerifier [0x0076EB81+569713]\nE       \tGetHandleVerifier [0x0076DD76+566118]\nE       \tOrdinal0 [0x00670B2B+1968939]\nE       \tOrdinal0 [0x00675988+1989000]\nE       \tOrdinal0 [0x00675A75+1989237]\nE       \tOrdinal0 [0x0067ECB1+2026673]\nE       \tBaseThreadInitThunk [0x75906739+25]\nE       \tRtlGetFullPathName_UEx [0x77408E7F+1215]\nE       \tRtlGetFullPathName_UEx [0x77408E4D+1165]\n\nD:\\python\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py:89: TimeoutException"}, "steps": [{"name": "添加失败截图...", "status": "passed", "attachments": [{"name": "失败截图", "source": "41812197-fd41-42ee-8dce-c5adb03f3e0d-attachment.png", "type": "image/png"}], "start": 1651826050671, "stop": 1651826050850}], "attachments": [{"name": "stdout", "source": "eaada2e7-cbc9-4c33-a415-8da9c18f91a5-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "Parameter", "value": "{'case_no': '', 'casename': '使用Wrong order inform 提交退货单成功', 'mode': '订单管理', 'data': 'reason=Wrong order inform ation\\ntext=123412412\\nimg=C:\\\\Users\\\\admin\\\\PycharmProjects\\\\pythonProject\\\\okmarts_ui_test\\\\data\\\\1.jpg', 'assert_way': 'xpath=/html/body/div[2]/span/div/div/div/span', 'result': 'success'}"}], "start": 1651825993175, "stop": 1651826050649, "uuid": "7ce78361-c95c-4f54-aeb5-6f2f36fb5bf7", "historyId": "36a3cec602bf5e5bc7fdc4d8619e0730", "testCaseId": "7ada25215fe1778912de4f80f444a520", "fullName": "test_main.Test_Main#test_my_order", "labels": [{"name": "feature", "value": "订单管理"}, {"name": "suite", "value": "test_main"}, {"name": "subSuite", "value": "Test_Main"}, {"name": "host", "value": "DESKTOP-NB3FDE0"}, {"name": "thread", "value": "22192-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_main"}]}